lib/danger/request_sources/github.rb

```diff
@@ -1,5 +1,6 @@
 require 'rest'
 require 'json'
+require 'base64'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.